### PR TITLE
reduced login redundancy

### LINKED ISSUE #4456: Combine app/views/user_sessions/_new.html.erb and new.html.erb
These two files are very similar except for the form url. Please keep new.html.erb but have it render _new.html.erb as a partial. 

We can figure out which url to use possibly by passing a local into the partial.

### PR DESCRIPTION
**Pull request by Dsxv**
Fixes #4456   (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] PR is descriptively titled 📑
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!
updated PR version of https://github.com/publiclab/plots2/pull/4580

### CODE REVIEW

**Comment by SidharthBansal** [comment]
Please provide new gifs for
1. Login modal
2. SignUp modal
3. Login page
4. Signup page
5. Edit Profile page

File: app/views/user_sessions/new.html.erb [hunk 1]
```diff
@@ -1,47 +1,66 @@
...
-
-  
   <br style="clear:both;"/>
 
-  <div class="form-group">
-
-    <label for="username"><%= t('user_sessions.new.username') %></label>
-    <%= f.text_field :username, { tabindex: 1, class: 'form-control', id: 'username-signup' } %>
-
+  <div class='col-lg-11 col-md-11 col-sm-11' style="display:flex; justify-content: center;">
```
Comment (SidharthBansal) [inline app/views/user_sessions/new.html.erb]: Are these changes related to your issue?

Comment (Dsxv) [reply app/views/user_sessions/new.html.erb]: I've actually renamed the partial _new.html.erb to file new.html.erb and deleted partial _new.html.erb , and changed the header file , 
The thing is that it should have shown the whole file (full green) to change however many things were common in both file and partial that's why its showing like this

Comment (SidharthBansal) [reply app/views/user_sessions/new.html.erb]: Can you please adjust it a little bit so that it will easily show up your changes?
It is hard to deduce what you did. I think you will be able to do it in 2-3 commits.
Thanks

Comment (Dsxv) [reply app/views/user_sessions/new.html.erb]: Sure !

Comment (Dsxv) [reply app/views/user_sessions/new.html.erb]: @SidharthBansal  should I revert the commits and do them again one by one ?

Comment (SidharthBansal) [reply app/views/user_sessions/new.html.erb]: As you wish. Just make it a little more readable.

**Comment by SidharthBansal** [comment]
Please provide the 5 gifs with these changes.

**Comment by SidharthBansal** [comment]
As a told you earlier. Provide the new gifs.

**Comment by jywarren** [comment]
This is nicely done. Thanks a lot!!!
